picby_app/src/navigation/Sidebar.tsx

```diff
@@ -10,14 +10,21 @@ import {globalStyles} from '../common/styles/globalStyles';
 import {menuColors} from '../staticData/staticData';
 import {TouchableOpacity} from 'react-native-gesture-handler';
 import Header from './Header';
+import {NavigationRoute, NavigationParams} from 'react-navigation';
 
 const {YELLOW_COLOR} = menuColors;
 const {width: vw} = Dimensions.get('window');
 
+const navItemsWithoutOne = (array: NavigationRoute<NavigationParams>[]) => {
```

Comment:
ta funkcja nie działą na stanie ani komponencie. Powinna być w oddzielnym module `.utils.ts` dla danego widoku. Dodatkowo powinna przyjmować klucz do porównań jako parametr a nie mieć go zahardkodowanego wewnatrz.

Reply:
i chyba powinna mieć lepszą nazwę w stylu `omitNavItems`. Dodatkowo nazwa parametru jako array jest zbyt ogólna. Raczej `navItems`. Dodatkowo jeśli funkcja przyjmuje więcej niż jeden parametr to powinna przyjmować obiekt tak żeby móc użyć wzorca named arguments w JS.

Reply:
Poza tym czy tak naprawdę do SideBaru nie powinny już być przekazywane właściwe wyfiltrowane navItems? Tak żeby ten komponent nie musiał się już zajmować filtrowaniem